python/ecl/summary/ecl_sum_vector.py

```diff
@@ -99,8 +100,20 @@ def days(self):
     def mpl_dates(self):
         """
         All the dates as numpy vector of dates in matplotlib format.
+        This property will be replaced by numpy_dates, but is kept for
+        backwards-compatibility for the time-being. Usage will trigger
+        a depreciation warning.
         """
-        return self.__mpl_dates
+        warning.warn("The mpl_dates property has been deprecated - use numpy_dates instead",
+                     DeprecationWarning)
+        return self.__numpy_dates
+
+    @property
+    def numpy_dates(self):
+        """
+        All the dates as numpy vector of dates in matplotlib format.
```

Comment:
Are you sure that this is matplotlib format? When looking at the parent implementation it really seems to be `numpy` format..